packages/bigtable/test/index.js

```diff
@@ -184,21 +193,45 @@ describe('Bigtable', function() {
       assert.strictEqual(calledWith.baseUrl, endpoint);
       assert.strictEqual(calledWith.customEndpoint, true);
 
-      Object.keys(calledWith.protoServices).forEach(function(service) {
-        service = calledWith.protoServices[service];
+      Object.keys(bigtable.protos).forEach(function(protoKey) {
+        var proto = bigtable.protos[protoKey];
 
-        if (is.object(service)) {
-          assert.strictEqual(service.baseUrl, endpoint);
+        if (is.object(proto)) {
+          assert.strictEqual(proto.baseUrl, endpoint);
+        }
+      });
+
+      delete process.env.BIGTABLE_EMULATOR_HOST;
+    });
+
+    it('should use GOOGLE_CLOUD_BIGTABLE_ENDPOINT if defined', function() {
```

Comment:
This test isn't needed. We only need to check that the correct environment variables were passed to the Service constructor.